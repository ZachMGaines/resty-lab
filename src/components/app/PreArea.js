import React from 'react';


const preArea = ({ body }) => {
  return <pre>{JSON.stringify(body, null, 2)}</pre>;
};

export default preArea;
