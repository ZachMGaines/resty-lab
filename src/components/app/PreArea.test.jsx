import React from 'react';
import { render } from '@testing-library/react';
import PreArea from './PreArea';

describe('PreArea component', () => {
  it('renders pre area', () => {
    const { asFragment } = render(
      <PreArea
        body="name: none"
      />
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
