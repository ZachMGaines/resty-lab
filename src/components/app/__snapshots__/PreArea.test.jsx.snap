// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PreArea component renders pre area 1`] = `
<DocumentFragment>
  <pre>
    "name: none"
  </pre>
</DocumentFragment>
`;
